src/test/java/org/sagebionetworks/template/SesClientImplTest.java

```diff
@@ -0,0 +1,69 @@
+package org.sagebionetworks.template;
+
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService;
+import com.amazonaws.services.simpleemail.model.SetIdentityNotificationTopicRequest;
+import com.amazonaws.services.simpleemail.model.SetIdentityNotificationTopicResult;
+import net.bytebuddy.asm.Advice;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SesClientImplTest {
+
+    SesClientImpl client;
```

Comment:
private

Reply:
@InjectMocks should automatically create an instance with the constructor mocked dependencies